scenario/utils.ts

```diff
@@ -254,3 +266,178 @@ export async function fetchLogs(
     return contract.queryFilter(filter, fromBlock, toBlock);
   }
 }
+
+export async function executeOpenProposal(
+  dm: DeploymentManager,
+  { id, startBlock, endBlock }: OpenProposal
+) {
+  const governor = await dm.getContractOrThrow('governor');
+  const blockNow = await dm.hre.ethers.provider.getBlockNumber();
+  const blocksUntilStart = startBlock - blockNow;
+  const blocksUntilEnd = endBlock - Math.max(startBlock, blockNow);
+
+  if (blocksUntilStart > 0) {
+    await mineBlocks(dm, blocksUntilStart);
+  }
+
+  const compWhales = dm.network === "mainnet" ? COMP_WHALES.mainnet : COMP_WHALES.testnet;
```

Comment:
nit: single quotes

Reply:
Also, this logic is duplicated in `CometContext.getCompWhales()`, but I see we're not passing in context here so it's probably easier just to leave it as is

Reply:
updated